packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -82,6 +88,17 @@ export const DetailView: FC = () => {
 
       <Footer draft={draft} save={save} />
       <SidePanel draft={draft} />
+
+      <Modal
+        title={t('heading.add-item')}
+        cancelButton={<DialogButton variant="cancel" onClick={hideDialog} />}
+        nextButton={<DialogButton variant="next" onClick={() => {}} />}
+        okButton={<DialogButton variant="ok" onClick={hideDialog} />}
+        height={600}
+        width={900}
+      >
+        <InboundLineEdit />
+      </Modal>
```

Comment:
This could be silly but thought i'd throw this into the detail view for now. It might make disabling the buttons more difficult - but we're only checking if quantities are zero to disable it. And actually.. maybe we should be allowing that? What's the point of disabling it 🤔

Reply:
happy with either option. the outbound was moved out in order to control the validation and disable the OK, but that process has changed now.
Might not be required - I just wonder if it will be strange if the server strips out or deletes 0 quantity lines, which I think it was going to do? I don't see any value in adding a zero quantity item; it provides another way to remove an existing line which I think is more intuitive than the bulk action - but the comment has also been made that we don't want multiple ways of doing the same thing.
Overall I'd prefer to keep the validation - would be interested what others think though

Reply:
I think I heard someone mention that zero stock lines should be removed 🤔  so.. should be nothing to prune